### DISCUSSION TITLE
PKCE support for OAuth2/OIDC

### DISCUSSION THREAD

**Discussion by AlexeyVT** [requarks/wiki#6384] [Help / Questions]
When setting up generic oauth2/oidc, I don't find PKCE support. The server requires the mandatory use of PKCE, which is why it is impossible to make friends with wiki.js and generic OAuth2. Am I dumb, or is there really no PKCE support in the module?

**Comment by chaitanya-enmovil** [comment]
Hey, did you get to resolve the issue? I'm facing the same issue. I tried adding pkce: true in authentication.js but wiki immediately throws somthing went wrong error!
